### PR TITLE
add incremental pipeline

### PR DESCRIPTION
**Pull request by kristof-bb**

### CODE REVIEW

**Comment by abbas123456** [comment]
The regex used in the `__getattr__` method of `app.etl.etl_base.DataPipeline` needs to be changed from:

`pat = re.compile('_l([0-9])_table')`

to 

`pat = re.compile('_l([0-9])_(sequence|table)')`

This is to allow the new sequence properties to work

File: app/etl/etl_incremental_data.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        else:  # append as is
+            self.dbi.append_table(self._l0_temp_table, self._l0_table, drop_source=True)
+        self.dbi.append_table(source_table=self._l1_temp_table, target_table=self._l1_table)
+
+    def _create_table(self, table_name, column_types, drop_existing=False):
+        if drop_existing:
+            self.dbi.drop_table(table_name)
+        columns = ','.join(f'{c} {t}' for c, t in column_types)
+        stmt = f'CREATE TABLE IF NOT EXISTS {table_name} ({columns})'
+        self.dbi.execute(stmt)
```
Comment (abbas123456) [inline app/etl/etl_incremental_data.py]: Should be `self.dbi.execute_statement(stmt)`

File: app/etl/etl_incremental_data.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+    def _create_sequence(self, sequence_name):
+        stmt = f'CREATE SEQUENCE IF NOT EXISTS {sequence_name}'
+        self.dbi.execute(stmt)
```
Comment (abbas123456) [inline app/etl/etl_incremental_data.py]: Same as above

File: app/etl/etl_incremental_data.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        stmt = f"""
+            INSERT INTO {self._l1_temp_table}
+            (
+                {column_name_string}
+            )
+            SELECT
+                {selection}
+            FROM {self._l0_temp_table}
+        """
+        self.dbi.execute(stmt)
```
Comment (abbas123456) [inline app/etl/etl_incremental_data.py]: Same as above

File: app/etl/etl_incremental_data.py [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+        stmt = f"""
+            INSERT INTO {self._l0_table}
+            (
+                {column_name_string}
+            )
+            SELECT
+                {selection}
+            FROM {self._l0_temp_table}
+        """
+        self.dbi.execute(stmt)
```
Comment (abbas123456) [inline app/etl/etl_incremental_data.py]: Same as above

File: app/commands/dev/datafiles_to_db_by_source.py [hunk 5]
```diff
@@ -16,10 +16,10 @@
 }
 
 
-@click.command('datafiles_to_events_by_source')
+@click.command('datafiles_to_db_by_source')
```
Comment (marcuspp) [inline app/commands/dev/datafiles_to_db_by_source.py]: You will need to update  jenkins as they call this command in task runners

**Pull request merged**